File: examples-sdk/python/503-villog-multi-drain.py
#!/usr/bin/env python3
"""503-villog-multi-drain — Python SDK equivalent
Compile: vil compile --from python --input 503-villog-multi-drain.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("app", port=8080)
server.compile()

File: examples-sdk/typescript/503-villog-multi-drain.ts
#!/usr/bin/env tsx
// 503-villog-multi-drain — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 503-villog-multi-drain.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("app", 8080);
server.compile();
